src/components/provider-list-item/provider-list-item.js

```diff
@@ -36,23 +38,41 @@ function ProviderListItem({ item, link, active, onClick, headingLevel }) {
         {item.name}
       </Headline>
 
-      <div data-test-eholdings-provider-list-item-selections>
-        <span data-test-eholdings-provider-list-item-num-packages-selected>
-          <FormattedNumber value={item.packagesSelected} />
-        </span>
-
-        &nbsp;/&nbsp;
-
-        <span data-test-eholdings-provider-list-item-num-packages-total>
-          <FormattedNumber value={item.packagesTotal} />
-        </span>
-
-        &nbsp;
+      <div
+        data-test-eholdings-provider-list-item-selections
+        className={cx('itemMetadata')}
+      >
+        <AppIcon
+          app={APP_ICON_NAME}
+          iconKey='selectedPackage'
+          size='small'
+          className={cx('item', 'selection-status', {
+            'not-selected': !item.packagesSelected,
+          })}
+        >
+          <span data-test-eholdings-package-list-item-selected>
+            {item.packagesSelected ?
```

Comment:
Could it be formatted like this?

```suggestion
            {item.packagesSelected
            ? (<FormattedMessage
                id="ui-eholdings.selectedCount"
                values={{
                  count: (
                    <span data-test-eholdings-provider-list-item-num-packages-selected>
                      <FormattedNumber value={item.packagesSelected} />
                    </span>
                  )
                }}
              />)
            : (<FormattedMessage id="ui-eholdings.notSelected" />)}
```